SPECS/libdb/libdb.spec

```diff
@@ -8,6 +8,7 @@ Distribution:   Mariner
 Group:          System/Libraries
 URL:            https://oss.oracle.com/berkeley-db.html
 Source0:        http://download.oracle.com/berkeley-db/db-%{version}.tar.gz
+Patch0:         CVE-2019-2708.patch
```

Comment:
Double-checking: will our tooling catch all these `*.nopatch` files now? I see we tend to mention them in other specs (especially inside `kernel.spec`) but maybe that's no longer required.

Reply:
that's no longer required